ReactAndroid/src/main/java/com/facebook/react/modules/camera/CameraRollManager.java

```diff
@@ -222,43 +231,48 @@ public void getPhotos(final ReadableMap params, final Promise promise) {
     int first = params.getInt("first");
     String after = params.hasKey("after") ? params.getString("after") : null;
     String groupName = params.hasKey("groupName") ? params.getString("groupName") : null;
+    String assetType = params.hasKey("assetType") ? params.getString("assetType") : ASSET_TYPE_PHOTOS;
     ReadableArray mimeTypes = params.hasKey("mimeTypes")
         ? params.getArray("mimeTypes")
         : null;
     if (params.hasKey("groupTypes")) {
       throw new JSApplicationIllegalArgumentException("groupTypes is not supported on Android");
     }
 
-    new GetPhotosTask(
+    new GetMediasTask(
           getReactApplicationContext(),
           first,
           after,
           groupName,
+          assetType,
           mimeTypes,
           promise)
           .executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
   }
 
-  private static class GetPhotosTask extends GuardedAsyncTask<Void, Void> {
+  private static class GetMediasTask extends GuardedAsyncTask<Void, Void> {
     private final Context mContext;
     private final int mFirst;
     private final @Nullable String mAfter;
     private final @Nullable String mGroupName;
+    private final String mAssetType;
     private final @Nullable ReadableArray mMimeTypes;
     private final Promise mPromise;
 
-    private GetPhotosTask(
+    private GetMediasTask(
```

Comment:
Change to `GetMediaTask`. I'm pretty sure that one `medium`, multiple `medias`. Check: https://english.stackexchange.com/questions/314099/should-i-use-media-or-medias

Reply:
I think here we get ```medias``` - multiple. Isn't it? Previously we get ```photos```. What's wrong here?

Reply:
Media is already plural, the singular is medium.

Reply:
Ok, Got it